composer.json

```diff
@@ -23,12 +23,14 @@
         "twig/twig": "~1.12"
     },
     "require-dev": {
-        "sonata-project/user-bundle": "~2.2",
```

Comment:
* Not related to Travis.
* :-1: Anyway, it's a dev require for test only. This should be kept.